target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -533,6 +534,7 @@ ramips_setup_macs()
 	whr-1166d|\
 	whr-300hp2|\
 	whr-600d|\
+	whr-g300n|\
```

Comment:
That doesn't look entirely correct to me. If the lan mac is already set via the dts, it shouldn't be necessary to set it here once more. A section:

```
whr-g300n)
 	wan_mac=$(mtd_get_mac_binary factory 4)
	;;
```

should be enough.

Am I right that without a WAN mac address, the LAN mac address incremented by one is used for wan?

Reply:
You are right, "lan_mac" is unnecessary...

I will fix this PR.

Reply:
Would you mind to answer/test for my other question as well:

> Am I right, that without a WAN mac address, the LAN mac address incremented by one is used for wan?

Reply:
Oh, sorry. I forgot to confirm about that...
I'll do it.

Reply:
Confirmed. You were right about "wan_mac" as well...
I'll fix this PR again.

Reply:
It raises the next question.

Initially you used the same mac for lan and wan. Now you are using lan+1 for wan. Is it intentional? Is the stock firmware doing the same (is the incremented lam mac reserved for this board)?

Reply:
I confirmed with stock firmware, the same MAC address was used for WAN and LAN.

In the commits up to the present, I have written incorrect configurations by speculation due to lack of understanding and confirmation about MAC address...

I will reflect on and fix again to the correct state.